felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/loeser/ObjectRiver.kt

```diff
@@ -86,7 +86,7 @@ abstract class ObjectRiver<Melding : Any> {
      * Utgående melding som skal publiseres når innkommende melding er ferdig prosessert.
      * Returneres '`null`' så vil ingen utgående melding publiseres.
      */
-    protected abstract fun Melding.haandter(): Map<Key, JsonElement>?
+    protected abstract fun Melding.haandter(json: Map<Key, JsonElement>): Map<Key, JsonElement>?
```

Comment:
For å bruke `json` i en eventuell `Fail`. Ikke kjempefornøyd med endringen, men kom ikke på noe bedre i farten.